torch/csrc/jit/tensorexpr/loopnest.cpp

```diff
@@ -1089,35 +1089,56 @@ void LoopNest::splitWithMask(For* f, int factor, For** outer, For** inner) {
   // TODO: record history of transformations
 }
 
-void LoopNest::reorderAxis(Tensor* t, For* a, For* b) {
+For* findOuterFor(For* a, For* b) {
```

Comment:
Does this need to handle `a == b`?

Reply:
We handle it in reorderAxis before calling, and this isn't a publically available func.